coral.py

```diff
@@ -838,5 +843,6 @@ def draw_grid():
 
     # Update display and move clock.
     pygame.display.update()
+    print(snake.oranges)
```

Comment:
please remove the unnecessary print

Reply:
oops, forgot to remove that while testing

Reply:
added orange giving energy. the toggle option is a bit harder however: the configuration menu code is somewhat hard to understand.